phpunit.xml

```diff
@@ -9,6 +9,9 @@
             <directory suffix=".php">./src</directory>
         </whitelist>
     </filter>
+    <php>
+        <env name="DB_CONNECTION" value="testing"/>
```

Comment:
Where's this connection defined? Looks handy.

Reply:
I am not use to be honest. [The `testing` connection is mentioned in the techbench documentation](https://github.com/orchestral/testbench#overriding-setup-method).